pkgs/top-level/all-packages.nix

```diff
@@ -36901,4 +36901,6 @@ with pkgs;
   swift-corelibs-libdispatch = callPackage ../development/libraries/swift-corelibs-libdispatch { };
 
   swaysettings = callPackage ../applications/misc/swaysettings { };
+
+  pizarra = callPackage ../applications/graphics/pizarra { };
```

Comment:
Please sort this into a fitting category.

Reply:
What would the fitting category be?
I thought it was a graphics application, and that's why I put it there.

Reply:
Oh, inside of the file.
To be completely honest, I didn't even see the grouping. My bad.